rxandroidble/src/test/groovy/com/polidea/rxandroidble2/internal/RxBleDeviceProviderTest.groovy

```diff
@@ -39,6 +39,13 @@ class RxBleDeviceProviderTest extends Specification {
         io.reactivex.Observable<RxBleConnection> establishConnection(boolean autoConnect, Timeout operationTimeoutSetup) {
             establishConnection(autoConnect)
         }
+
+//        @Override
```

Comment:
I suppose that the `@Override` shouldn't be commented out. It applies to the method below which is related to the new API which is not yet public.